### ISSUE TITLE
[F glutil.cpp:338] eglInitialize() failed

### ISSUE THREAD

**Issue by adavies25** [nv-tlabs/GET3D#135]
-----
Hi, thank you for writing this software. 

I first tried to use the google collab file here:
[https://colab.research.google.com/drive/1AAE4jp39rXhW2zmlNwpWkvDPULugIXfk?usp=sharing](https://colab.research.google.com/drive/1AAE4jp39rXhW2zmlNwpWkvDPULugIXfk?usp=sharing)
and got this same error eglInitialize() failed when running inference. I installed "libnvidia-gl-535" and it fixed that problem and generated a mesh.

Now I'm trying to use the Dockerfile on Windows with a GeForce RTX 3090 card and I'm getting the same error again. Installing libnvidia-gl-535 didn't help this time, so I'm not sure what to try next. Here is the run and output:

Creating output directory...
Launching processes...
Setting up PyTorch plugin "upfirdn2d_plugin"... Done.
Setting up PyTorch plugin "bias_act_plugin"... Done.
Setting up PyTorch plugin "filtered_lrelu_plugin"... Done.
==> resume from pretrained path ./get3d_release/shapenet_motorbike.pt
==> generate 
/workspace/training/networks_get3d.py:430: UserWarning: torch.range is deprecated and will be removed in a future release because its behavior is inconsistent with Python's range builtin. Instead, use torch.arange, which produces values in [start, end).
  camera_theta = torch.range(0, n_camera - 1, device=self.device).unsqueeze(dim=-1) / n_camera * math.pi * 2.0
[F glutil.cpp:338] eglInitialize() failed
./genmesh.sh: line 1:   343 Aborted                 python train_3d.py --outdir=save_inference_results/shapenet_motorbike --gpus=1 --batch=4 --gamma=40 --data_camera_mode shapenet_motorbike --dmtet_scale 1.0 --use_shapenet_split 1 --one_3d_generator 1 --fp32 0 --inference_vis 1 --resume_pretrain ./get3d_release/shapenet_motorbike.pt --inference_to_generate_textured_mesh 1

Any help would be appreciated. Thank you.
Alan

**Comment by Bathsheba** [comment]
https://github.com/nv-tlabs/GET3D/issues/115

**Comment by adavies25** [comment]
> #115

Thanks works!
